### PR TITLE
LPS-161439 Cover ObjectEntryEntityModel by using integration tests to avoid break changes

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**

### CODE REVIEW

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/odata/entity/v1_0/test/ObjectEntryEntityModelTest.java [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+			_createObjectField(ObjectFieldConstants.DB_TYPE_LONG),
+			new ObjectFieldBuilder().relationshipType(
+				ObjectRelationshipConstants.TYPE_ONE_TO_MANY
+			).name(
+				"oneToManyObjectFieldName"
+			).build(),
+			_createObjectField(ObjectFieldConstants.DB_TYPE_STRING));
+
+		_expectedEntityFieldsMap.putAll(
+			Stream.of(
```
Comment (marcelabc) [inline modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/odata/entity/v1_0/test/ObjectEntryEntityModelTest.java:125]: Unfortunately, could you try to exchange the usage of Stream to For please?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi @MarinhoFeliphe ,
I just pointed out the change of the Stream. Besides that, LGTM!! :tada:

**Comment by marcelabc** [comment]
@MarinhoFeliphe rebase error :confused:

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**